library/core/src/intrinsics.rs

```diff
@@ -2472,6 +2485,7 @@ pub const unsafe fn write_bytes<T>(dst: *mut T, val: u8, count: usize) {
 #[rustc_const_unstable(feature = "const_eval_select", issue = "none")]
 #[lang = "const_eval_select"]
 #[rustc_do_not_const_check]
+#[track_caller]
```

Comment:
Does this do anything? The intrinsic disappears before the attribute has any effect, I think.
```suggestion
// The function is compiled away, so track_caller has no effect, but merely
// serves as documentation that caller location information is propagated.
#[track_caller]
```

Reply:
I left it while debugging. there is `cfg(bootstrap)` above as we have now switched to the intrinsic so I am not worried about this

Reply:
Oh, I didn't realize this is the old function.

Seems better not to change it, then. Who knows if that breaks something...
```suggestion
```